src/app/tests/suites/certification/Test_TC_S_2_2.yaml

```diff
@@ -431,6 +431,21 @@ tests:
               - name: "SceneID"
                 value: 0x01
 
+    - label:
+          "Step 4e: TH sends a RecallScene command to DUT with the GroupID field
+          set to GI (Where GI is a group currently absent from the group table)
+          and the SceneID field set to 0x01."
+      PICS: S.S.C05.Rsp
```

Comment:
total nit, but this command is mandatory, so it is probably better to just omit the PICS here - that way you know the step is going to be run on all devices regardless of whether or not the pics is set correctly. The pics file for the CI in particular tends to be improperly defined and is easy to break by accident.